cmd/build/helmify/static/README.md

```diff
@@ -62,63 +62,68 @@ _See [Exempting Namespaces](https://open-policy-agent.github.io/gatekeeper/websi
-| secretAnnotations                            | The annotations to add to the Gatekeeper secrets                                       | `{}`                                                                      |
-| pdb.controllerManager.minAvailable           | The number of controller manager pods that must still be available after an eviction   | `1`                                                                       |
-| service.type                                 | Service type                                                                           | `ClusterIP`                                                               |
-| service.loadBalancerIP                       | The IP address of LoadBalancer service                                                 | ``                                                                        |
-| rbac.create                                  | Enable the creation of RBAC resources                                                  | `true`                                                                    |
+| Parameter                                     | Description                                                                            | Default                                                                   |
+| :-------------------------------------------- | :------------------------------------------------------------------------------------- | :------------------------------------------------------------------------ |
+| postInstall.labelNamespace.enabled            | Add labels to the namespace during post install hooks                                  | `true`                                                                    |
+| postInstall.labelNamespace.image.repository   | Image with kubectl to label the namespace                                              | `openpolicyagent/gatekeeper-crds`                                         |
+| postInstall.labelNamespace.image.tag          | Image tag                                                                              | Current release version: `v3.7.0`                                         |
+| postInstall.labelNamespace.image.pullPolicy   | Image pullPolicy                                                                       | `IfNotPresent`                                                            |
+| postInstall.labelNamespace.image.pullSecrets  | Image pullSecrets                                                                      | `[]`                                                                      |
+| preUninstall.deleteWebhooks.enabled           | Delete webhooks before gatekeeper itself is uninstalled                                | `false`                                                                   |
```

Comment:
```suggestion
| preUninstall.deleteWebhooks.enabled           | Delete webhook configs before gatekeeper itself is uninstalled                                | `false`                                                                   |
```

Reply:
Thanks for the review. I applied the suggestion and rebased on latest master.